Makefile

```diff
@@ -48,7 +49,10 @@ deps: gopath
 
 .PHONY: validate
 validate:
-	@./tests/validate/gofmt.sh
+	# Run gofmt only on version 1.10
+	ifneq (,$(findstring 1.10,$(GOVERSION)))
```

Comment:
We should only run this on 1.11?

Reply:
gofmt I thought was failing on 1.11, not 1.10, but regardless, I'm still getting a syntax error on this line.

Reply:
Can we fix our code to confirm with what golang 1.11 wants, or is golang 1.11 just broken?